homeassistant/components/sleepiq.py

```diff
@@ -39,7 +39,7 @@
 DATA = None
 
 CONFIG_SCHEMA = vol.Schema({
-    DOMAIN: vol.Schema({
+    vol.Required(DOMAIN): vol.Schema({
```

Comment:
Is this a new standard? I haven't seen any other component that does it like this.